examples/tests_ci/run_test_sink_synchro.py

```
@@ -0,0 +1,202 @@
+    dr = 0.05
+    bmin = (-0.6, -0.6, -0.6)
+    bmax = (0.6, 0.6, 0.6)
+    model.resize_simulation_box(bmin, bmax)
+
+    setup = model.get_setup()
+    gen = setup.make_generator_lattice_hcp(dr, bmin, bmax)
+    setup.apply_setup(gen)
+
+    eng = shamrock.algs.gen_seed(42)
+
+    def vel_func(r):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `vel_func`

(ANN202)